web-sdk/v3.md

```diff
@@ -628,3 +628,156 @@ Here are some examples of what is achievable using the customization:
     { buttonText: 'Theme 2', slug: 'OJbxZzW', title: 'V3 Fidel Web SDK' },
     { buttonText: 'Theme 3', slug: 'eYBGrrQ', title: 'V3 Fidel Web SDK' },
 ]} />
+
+## Card entity
+
+This object is returned through the `onCardEnrolledCallback` when a card is successfully enrolled.
+
+<table>
+<thead>
+  <tr>
+    <th>Parameter</th>
+    <th>Value</th>
+    <th>Description</th>
+</thead>
+<tbody>
+  <tr>
+    <td><code>id</code></td>
+    <td>string</td>
+    <td>ID of the enrolled card</td>
+  </tr>
+  <tr>
+    <td><code>accountId</code></td>
+    <td>string</td>
+    <td>Account ID of the program where the card was enrolled</td>
+  </tr>
+  <tr>
+    <td><code>programId</code></td>
+    <td>string</td>
+    <td>ID of the program where the card was enrolled</td>
+  </tr>
+  <tr>
+    <td><code>scheme</code></td>
+    <td>string<br>Possible values: 'mastercard', 'visa', 'amex'</td>
+    <td>Card scheme</td>
+  </tr>
+  <tr>
+    <td><code>lastNumbers</code></td>
+    <td>number</td>
+    <td>Last 4 numbers of the card</td>
+  </tr>
+  <tr>
+    <td><code>firstNumbers</code></td>
+    <td>number</td>
```

Comment:
It's a string.  That's because it can be obfuscated:
1. '123456'
2. '*******' 

In the future we might even have partial first digits offuscation.